src/features/device-provisioning/register.ts

```diff
@@ -79,6 +80,7 @@ export const register: RequestHandler = async (req, res) => {
 			const device = await resinApiTx.post({
 				resource: 'device',
 				body: {
+					// @ts-expect-error This is for backwards compatibility :(
```

Comment:
In what sense?

Reply:
It's not generically typed but I've added a single commit so you can see the somewhat hardcoded version that works for what we have